lucille-examples/lucille-joining-db-connector-example/scripts/run_ingest.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+java -Dconfig.file=conf/joining-db-connector-example.conf -cp 'lib/*' com.kmwllc.lucille.core.Runner -local
```

Comment:
where is conf/joining-db-connector-example.conf in this PR?

Reply:
missing...  I'll add an example.